packages/tooling/cfp-template/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@microsoft/cfp-template",
```

Comment:
should the package name include "FAST"? All of our other packages do?

Reply:
Other parts of the CLI will break that naming convention due to how `npx`, which we plan to support, works. This makes it easier if we want to do any kind of find and replace for templates in the future so it sets a standard name.

Reply:
@janechu how does using our 'fast-' naming convention break `npx`?  I'd like to understand this more.  Changing our naming strategy inhibits discoverability when searching NPM for FAST packages.